test/index.js

```diff
@@ -23,7 +23,7 @@ describe('Forwarded', function () {
   it('should throw an exception when invalid request is passed', function (done) {
     var params
 
-    /*eslint-disable no-wrap-func */
+    /*eslint-disable no-extra-parens */
```

Comment:
The `no-wrap-func` rule in eslint was deprecated and replaced with `no-extra-parens`.